### PR TITLE
Localization implementation

### PR DESCRIPTION
**Pull request by Piteryo**
Added localization everywhere except menu (it is implemented in another pull request (androidBottomBar))

### CODE REVIEW

File: DotNetRu.Android/Helpers/LocaleUtils.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+using Android.Views;
+using Android.Widget;
+using Java.Util;
+
+namespace DotNetRu.Droid.Helpers
+{
+    public class LocaleUtils
+    {
+
+        private static Locale sLocale;
```
Comment (pfedotovsky) [inline DotNetRu.Android/Helpers/LocaleUtils.cs]: Is it just a local variable to store current locale? If yes, please rename it to currentLocale.
BTW, why it's static?

File: DotNetRu.Android/Helpers/LocaleUtils.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            }
+        }
+
+        public static void updateConfig(Application app, Configuration configuration)
+        {
+            if (sLocale != null)
+            {
+                //Wrapping the configuration to avoid Activity endless loop
+                Configuration config = new Configuration(configuration);
+                // We must use the now-deprecated config.locale and res.updateConfiguration here,
```
Comment (pfedotovsky) [inline DotNetRu.Android/Helpers/LocaleUtils.cs]: We're not supporting API level 17, is it code needed?

File: DotNetRu.Android/Localize.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                }
+                catch (CultureNotFoundException e2)
+                {
+                    // iOS language not valid .NET culture, falling back to English
+                    ci = new System.Globalization.CultureInfo("en");
+                }
+            }
+            return ci;
+        }
+        string AndroidToDotnetLanguage(string androidLanguage)
```
Comment (pfedotovsky) [inline DotNetRu.Android/Localize.cs]: We're not supporting all of this languages, why do we need this code?
Just fall back to "en", if current locale is not "ru"

File: DotNetRu.Clients.Portable/ViewModel/ViewModelBase.cs [hunk 4]
```diff
@@ -31,8 +41,17 @@ public class ViewModelBase : BaseViewModel
         public ViewModelBase(INavigation navigation = null)
         {
             this.Navigation = navigation;
+            Resources = new LocalizedResources(typeof(ApplicationResources.AppResources), CurrentLanguage);
         }
+        //public new void OnPropertyChanged([CallerMemberName]string property = null)
```
Comment (pfedotovsky) [inline DotNetRu.Clients.Portable/ViewModel/ViewModelBase.cs]: Can be deleted, I think?

File: DotNetRu.iOS/Localize.cs [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+                // ONLY use cultures that have been tested and known to work
+            }
+            return netLanguage;
+        }
+        string ToDotnetFallbackLanguage(PlatformCulture platCulture)
+        {
+            var netLanguage = platCulture.LanguageCode; // use the first part of the identifier (two chars, usually);
+            switch (platCulture.LanguageCode)
+            {
+                case "pt":
```
Comment (pfedotovsky) [inline DotNetRu.iOS/Localize.cs]: Removed unsupported locales here and above

File: DotNetRu.Android/Helpers/LocaleUtils.cs [hunk 6]
```diff
@@ -17,35 +17,32 @@ namespace DotNetRu.Droid.Helpers
     public class LocaleUtils
     {
 
-        private static Locale sLocale;
-        public static void setLocale(Locale locale)
+        private static Locale currentLocale;
```
Comment (pfedotovsky) [inline DotNetRu.Android/Helpers/LocaleUtils.cs]: Now I see that this is a duplicate of Locale.Default - why do we need it?

File: DotNetRu.Android/DotNetRu.Android.csproj [hunk 7]
```diff
@@ -15,7 +15,7 @@
     <AndroidResgenClass>Resource</AndroidResgenClass>
     <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
     <AndroidApplication>True</AndroidApplication>
-    <AndroidUseLatestPlatformSdk>true</AndroidUseLatestPlatformSdk>
+    <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
```
Comment (pfedotovsky) [inline DotNetRu.Android/DotNetRu.Android.csproj]: Why not use latest SDK?

File: DotNetRu.Clients.Portable/ViewModel/MeetupsViewModel.cs [hunk 8]
```diff
@@ -34,6 +38,7 @@ public class MeetupsViewModel : ViewModelBase
         public MeetupsViewModel(INavigation navigation)
             : base(navigation)
         {
+            MessagingCenter.Subscribe<LocalizedResources>(this, MessageKeys.LanguageChanged, sender => SortEvents());
```
Comment (pfedotovsky) [inline DotNetRu.Clients.Portable/ViewModel/MeetupsViewModel.cs]: Why do we need to sort events here?!

File: DotNetRu.Android/Localize.cs (hunk 3 above)
Comment (Piteryo) [reply DotNetRu.Android/Localize.cs]: For example, if user has such languages, there might be exception.

File: DotNetRu.Android/DotNetRu.Android.csproj (hunk 7 above)
Comment (bruce-willis) [reply DotNetRu.Android/DotNetRu.Android.csproj]: It's recommended to set compile version equal to target
It's guaranteed that all will build project using Android 8 => less problems (we all know ~~stability~~ of Xamarin)

File: DotNetRu.Android/DotNetRu.Android.csproj [hunk 9]
```diff
@@ -35,7 +35,7 @@
     <RunCodeAnalysis>false</RunCodeAnalysis>
     <AssemblyName>DotNetRu.Droid</AssemblyName>
     <EmbedAssembliesIntoApk>false</EmbedAssembliesIntoApk>
-    <AndroidSupportedAbis>armeabi-v7a</AndroidSupportedAbis>
+    <AndroidSupportedAbis>armeabi-v7a;x86</AndroidSupportedAbis>
```
Comment (pfedotovsky) [inline DotNetRu.Android/DotNetRu.Android.csproj]: Please create a new configuration for Emulator if you need to.
Debug builds for real device should be fast, we don't want to spent time for compiling for unneeded architecture

File: DotNetRu.Clients.Portable/ViewModel/MeetupsViewModel.cs (hunk 8 above)
Comment (pfedotovsky) [reply DotNetRu.Clients.Portable/ViewModel/MeetupsViewModel.cs]: @bruce-willis events are just meetups (so please rename the SortEvents) method - and they are sorted by date.

File: DotNetRu.Android/DotNetRu.Android.csproj (hunk 9 above)
Comment (bruce-willis) [reply DotNetRu.Android/DotNetRu.Android.csproj]: @pfedotovsky fixed

File: DotNetRu.Android/Helpers/LocaleUtils.cs [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+using Android.Views;
+using Android.Widget;
+using Java.Util;
+
+namespace DotNetRu.Droid.Helpers
+{
+    public class LocaleUtils
+    {
+
+        private static Locale currentLocale;
```
Comment (pfedotovsky) [inline DotNetRu.Android/Helpers/LocaleUtils.cs]: Is it a duplicate of Locale.Default?

File: DotNetRu.iOS/Localize.cs [hunk 11]
```diff
@@ -0,0 +1,91 @@
+﻿using System;
```
Comment (pfedotovsky) [inline DotNetRu.iOS/Localize.cs]: Same comments as for Android - remove unneeded locales

**Pull request merged**